packages/aws-cdk-lib/aws-codepipeline/lib/private/cross-region-support-stack.ts

```diff
@@ -150,6 +151,11 @@ export class CrossRegionSupportStack extends cdk.Stack {
   }
 }
 
-function generateStackName(props: CrossRegionSupportStackProps): string {
+function generateStackName(props: CrossRegionSupportStackProps): string | undefined {
+  // When the pipeline stack name is an unresolved token, we return 'undefined' here and
```

Comment:
I might be missing something, but I'm not sure how this method would return `undefined`.

Reply:
That was my previous solution, forgot to update the comment. Thanks